test/service/test_response.py

```diff
@@ -1397,5 +1397,32 @@ def test_summary_filter_none(self):
         self.assertEqual(summary_object['projectCount'], 2)
         self.assertEqual(summary_object['totalCellCount'], 6210.0)
 
+
+class TestIntegrationRepsonse(WebServiceTestCase):
```

Comment:
Better to have the word Portal in there to disambiguate from "integration test".

Reply:
WebServiceTestCase still starts up an ES container, unnecessarily so.

Reply:
Also typo in Repsonse